bio/example_test.go

```diff
@@ -22,7 +22,7 @@ func Example() {
 	// Poly can also output these file formats. Every file format has a corresponding Write function.
 	_ = gff.Write(gffInput, "test.gff")
 	_ = genbank.Write(gbkInput, "test.gbk")
-	_ = fasta.Write(fastaInput, "test.fasta")
+	_ = fasta.WriteFile(fastaInput, "test.fasta")
```

Comment:
why `WriteFile` instead of `Write`?

Reply:
`Write` in Golang is typically not used on paths (as far as I've seen), but on `io.Writer` interfaces. This change to `WriteFile` when using paths makes it more idiomatic. 

`Write` generally is also being moved into the bio package where we can more effectively test across all parser packages.